truss/templates/server/common/truss_server.py

```diff
@@ -83,6 +90,23 @@ class BasetenEndpoints:
 
     def __init__(self, model: ModelWrapper) -> None:
         self._model = model
+    # Create a Resource object to set the service name
+        resource = Resource(attributes={
+            ResourceAttributes.SERVICE_NAME: self._model._config.get("model_name").split("-")[0]
```

Comment:
This should be a property on the `ModelWrapper`. The splitting is a bit hacky, since somebody might legitly use `-` in the model name. For productionizing this, maybe make a regex that corresponds to how chains add a suffix and has a sensible fallback.